LinearModelling.Rmd

```diff
@@ -0,0 +1,1049 @@
+---
+title: "Introduction to Linear Modelling in R"
+output: 
+  learnr::tutorial:
+    css: "css/style.css"
+    progressive: true
+    allow_skip: true
```

Comment:
This is standard formatting from {learnr} styling. We can open an issue to change styling but would be better to handle as a general change rather than on a single course.

The black line next to 'What is modelling?' represents progress not selection. As a user moves through each page/activity included, the bar progresses down until it's full. As below, I have got to the end of the course, just before 'Review and feedback' but gone back to the 'Lines' section.

<img width="341" alt="Screenshot of course navigation showing progress bar" src="https://github.com/user-attachments/assets/279b2b7b-c527-493b-bf43-7dc2a3a8a50f">